style.css

```diff
@@ -317,3 +344,224 @@ span {
   border: 1px solid #7f8cff;
   border-radius: 8px;
 }
+
+.card-text {
+  padding-left: 3%;
+  padding-right: 3%;
+}
+
+@media (min-width: 768px) {
+  .nav-menu li {
+    display: block;
+    color: #344563;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    font-family: 'Poppins', sans-serif;
+  }
+
+  .heading {
+    padding-left: 260px;
+    padding-right: 260px;
+    padding-top: 40px;
+    background-image: url("images/header_bg.png");
+    background-repeat: no-repeat;
+    background-attachment: fixed;
+    background-position: 45px 25px !important;
+    background-size: 95%;
+  }
+
+  .social-links {
+    display: flex;
+    width: 20%;
+    justify-content: space-between;
+    padding-left: 0%;
+    gap: 9px;
+  }
+
+  .language-list li {
+    padding-right: 23px;
+  }
+
+  .social-links li {
+    list-style-type: none;
+  }
+
+  nav svg {
+    display: none;
+  }
+
+  nav {
+    padding-left: 10%;
+    padding-right: 10%;
+  }
+
+  .nav-menu {
+    display: flex;
+    gap: 33px;
+    list-style-type: none;
+  }
+
+  .card {
+    display: flex;
+    flex-direction: row;
+    width: 100%;
+    gap: 35px;
+  }
+
+  .card-text {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .works {
+    padding-left: 10%;
+    padding-right: 10%;
+  }
+
+  .card img {
+    width: 50%;
+    height: auto;
+  }
+
+  .tonic-items {
+    width: 60%;
+  }
+
+  .header-title {
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    color: #172b4d;
+    font-family: 'Poppins', sans-serif;
+    margin-bottom: 0;
+    margin-top: 1px;
+  }
+
+  .p-btn {
+    width: 30%;
+    padding-left: 12px;
+    padding-top: 12px;
+    padding-bottom: 12px;
+    color: #396df2;
+    letter-spacing: 0.03em;
+    background: #fff;
+    border: 1px solid #dfe1e6;
+    border-radius: 8px;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 24px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+  }
+
+  .sub-title {
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    color: #344563;
+    font-family: 'Poppins', sans-serif;
+  }
+
+  .card-l {
+    order: -4;
+  }
+
+  .card-l-2 {
+    order: -3;
+  }
+
+  .rv {
+    display: flex;
+    flex-direction: row-reverse;
+  }
+
+  .heading-lg {
+    display: flex;
+    flex-direction: row;
+    height: 90%;
+    justify-content: center;
+    gap: 33px;
+    padding-left: 0;
+    padding-right: 0;
+    background: #fff;
+    border-radius: 0 0 0 55px;
+    padding-bottom: 35px;
+  }
+
+  .about-p1 {
+    width: 35%;
+  }
+
+  .about-p2 {
+    width: 35%;
+  }
+
+  .language-list {
+    display: flex;
+    width: 55%;
+    justify-content: space-between;
+    padding-left: 0%;
+    flex-direction: row;
+    position: static;
+  }
+
+  .about-p1 button {
+    width: 35%;
+  }
+
+  .tech-list-f {
```

Comment:
- [ ] Kindly make sure the tech list looks like the Figma design.
 - Please change padding, fonts, and color so it looks like the Figma Design.

## Your Design:
![image](https://user-images.githubusercontent.com/92215878/193094624-6b2f160f-0c47-4ec6-8256-3d4a29dff4b9.png)

## Figma Design:
![image](https://user-images.githubusercontent.com/92215878/193094713-4b876a52-ab2b-429d-9257-8519429275f3.png)